app/javascript/packs/pages/admin/blogs.js

```diff
@@ -65,24 +66,84 @@ function BinxAdminBlogs() {
         submitOnSuccess: false,
         triggerUploadOnSubmit: false
       })
+      uppy.on('upload-success', (file, response) => {
+        this.publicImageTemplate(response.body.public_image)
```

Comment:
nit: I'd like the appending to the ul to happen here, and `publicImageTemplate` just return the string to render.